docs don't build with latest numpydoc

With the latest dev version of numpydoc the docs fail to build on master for me. Can anyone else reproduce this?

```shell
$ pip freeze | grep numpydoc
numpydoc==0.6.dev0
$ make -C doc clean html
rm -rf build/* source/generated
sphinx-build -b html -d build/doctrees  -W source build/html
Making output directory...
Running Sphinx v1.2.2
/Users/jairideout/miniconda/envs/scikit-bio/lib/python3.4/site-packages/matplotlib/__init__.py:1318: UserWarning:  This call to matplotlib.use() has no effect
because the backend has already been chosen;
matplotlib.use() must be called *before* pylab, matplotlib.pyplot,
or matplotlib.backends is imported for the first time.

  warnings.warn(_use_error_msg)
loading pickled environment... failed: [Errno 2] No such file or directory: '/Users/jairideout/dev/scikit-bio/doc/build/doctrees/environment.pickle'
[autosummary] generating autosummary for: alignment.rst, diversity.rst, draw.rst, index.rst, io.rst, sequence.rst, stats.rst, tree.rst, util.rst, workflow.rst
[autosummary] generating autosummary for: /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.alignment.Alignment.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.alignment.AlignmentError.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.alignment.AlignmentStructure.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.alignment.SequenceCollection.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.alignment.SequenceCollectionError.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.alignment.StripedSmithWaterman.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.alignment.TabularMSA.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.alignment.global_pairwise_align.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.alignment.global_pairwise_align_nucleotide.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.alignment.global_pairwise_align_protein.rst, ..., /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.util.create_dir.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.util.find_duplicates.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.util.flatten.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.util.get_data_path.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.util.is_casava_v180_or_later.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.util.remove_files.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.util.safe_md5.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.Workflow.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.method.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.requires.rst
[autosummary] generating autosummary for: /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.diversity.alpha.ace.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.diversity.alpha.berger_parker_d.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.diversity.alpha.brillouin_d.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.diversity.alpha.chao1.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.diversity.alpha.chao1_ci.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.diversity.alpha.dominance.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.diversity.alpha.doubles.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.diversity.alpha.enspie.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.diversity.alpha.esty_ci.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.diversity.alpha.faith_pd.rst, ..., /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.util.TestRunner.__format__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.util.TestRunner.test.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.Workflow.__call__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.Workflow.__format__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.Workflow.initialize_state.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.method.__call__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.method.__format__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.method.highest_priority.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.requires.__call__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/skbio.workflow.requires.__format__.rst
[autosummary] generating autosummary for: /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.io.registry.Format.__format__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.io.registry.Format.is_binary_format.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.io.registry.Format.monkey_patched_readers.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.io.registry.Format.monkey_patched_writers.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.io.registry.Format.name.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.io.registry.Format.reader.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.io.registry.Format.readers.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.io.registry.Format.sniffer.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.io.registry.Format.sniffer_function.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.io.registry.Format.writer.rst, ..., /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.stats.gradient.GradientANOVA.__format__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.stats.gradient.GradientANOVA.get_trajectories.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.stats.gradient.GradientANOVAResults.__format__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.stats.gradient.GradientANOVAResults.to_files.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.stats.gradient.GroupResults.__format__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.stats.gradient.GroupResults.to_files.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.stats.gradient.TrajectoryGradientANOVA.__format__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.stats.gradient.TrajectoryGradientANOVA.get_trajectories.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.stats.gradient.WindowDifferenceGradientANOVA.__format__.rst, /Users/jairideout/dev/scikit-bio/doc/source/generated/generated/skbio.stats.gradient.WindowDifferenceGradientANOVA.get_trajectories.rst
loading intersphinx inventory from http://docs.python.org/dev/objects.inv...
loading intersphinx inventory from http://docs.scipy.org/doc/scipy/reference/objects.inv...
loading intersphinx inventory from http://www.biom-format.org/objects.inv...
loading intersphinx inventory from http://pandas.pydata.org/objects.inv...
loading intersphinx inventory from http://matplotlib.org/objects.inv...
loading intersphinx inventory from http://docs.scipy.org/doc/numpy/objects.inv...
building [html]: targets for 14 source files that are out of date
updating environment: 677 added, 0 changed, 0 removed
reading sources... [  6%] generated/generated/skbio.io.registry.Format.__format__
Exception occurred:
  File "/Users/jairideout/miniconda/envs/scikit-bio/lib/python3.4/inspect.py", line 1890, in _signature_from_builtin
    raise ValueError("no signature found for builtin {!r}".format(func))
ValueError: no signature found for builtin <method '__format__' of 'object' objects>
The full traceback has been saved in /var/folders/h3/2l54lt8d69db92g2mlsc_wnw0000gp/T/sphinx-err-450p03ej.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
make: *** [html] Error 1
```

--- a/ci/pip_requirements.txt
+++ b/ci/pip_requirements.txt
@@ -5,5 +5,5 @@ contextlib2
 coveralls
 natsort
 CacheControl[FileCache]
-git+git://github.com/numpy/numpydoc.git
+git+git://github.com/numpy/numpydoc.git@1a848331c2cf53d4fe356f4607799524bcc577ed
 check-manifest

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -71,7 +71,8 @@ try:
 except ImportError:
     raise RuntimeError(
         "numpydoc v0.6 or later required. Install it with:\n"
-        "  pip install git+git://github.com/numpy/numpydoc.git")
+        "  pip install git+git://github.com/numpy/numpydoc.git@1a848331c2cf53"
+        "d4fe356f4607799524bcc577ed")
 
 @property
 def _extras(self):

